dApp/user/src/lib/Ethers/Ethers.res

```diff
@@ -48,41 +48,54 @@ type ethersBigNumber
 module BigNumber = {
   type t = ethersBigNumber
 
+  @module("ethers") @scope("BigNumber")
+  external fromUnsafe: string => t = "from"
+  @module("ethers") @scope("BigNumber")
+  external fromInt: int => t = "from"
+
   @send external add: (t, t) => t = "add"
   @send external sub: (t, t) => t = "sub"
   @send external mul: (t, t) => t = "mul"
   @send external div: (t, t) => t = "div"
+  @send external mod: (t, t) => t = "mod"
+  @send external pow: (t, t) => t = "pow"
+  @send external abs: t => t = "abs"
+
   @send external gt: (t, t) => bool = "gt"
+  @send external gte: (t, t) => bool = "gte"
   @send external lt: (t, t) => bool = "lt"
-  @dead("+eq") @send external eq: (t, t) => bool = "eq"
-  @send external cmp: (t, t) => int = "cmp"
-  @dead("+sqr") @send external sqr: t => t = "sqr"
+  @send external lte: (t, t) => bool = "lte"
+  @send external eq: (t, t) => bool = "eq"
+
   @send external toString: t => string = "toString"
-  @send external toStringRad: (t, int) => string = "toString"
 
   @send external toNumber: t => int = "toNumber"
   @send external toNumberFloat: t => float = "toNumber"
 }
 
+type providerType
+
 @send
-external waitForTransaction: (Web3.rawProvider, string) => JsPromise.t<txResult> =
-  "waitForTransaction"
+external waitForTransaction: (providerType, string) => JsPromise.t<txResult> = "waitForTransaction"
 
-type providerType
 type walletType = {address: string, provider: providerType}
 
 module Wallet = {
   type t = walletType
 
   @new @module("ethers")
   external makePrivKeyWallet: (string, providerType) => t = "Wallet"
+
+  type rawSignature
+  @send
+  external signMessage: (t, string) => JsPromise.t<rawSignature> = "signMessage"
 }
 
 module Providers = {
   type t = providerType
 
   @new @module("ethers") @scope("providers")
-  external makeProvider: string => Web3.rawProvider = "JsonRpcProvider"
+  external makeProvider: string => t = "JsonRpcProvider"
```

Comment:
Yeah, we should even delete the `Web3` module. And anything we find in it that we still use we should migrate to Ethers

Reply:
#34